buildconfig/CMake/QtTargetFunctions.cmake

```diff
@@ -75,6 +75,8 @@ endfunction()
 # keyword: INSTALL_DIR_BASE Base directory the build output. The final product goes into a subdirectory based on the Qt version.
 # keyword: OSX_INSTALL_RPATH Install path for osx version > 10.8
 function (mtd_add_qt_target)
+  set(CMAKE_CXX_CLANG_TIDY "") # disable clang-tidy for sip generated code
```

Comment:
This comment is not quite accurate. This is nothing to to with sip generation. It will in factor disable clang-tidy for all of the Qt code. Is that what we want?